core/src/main/kotlin/event/Event.kt

```diff
@@ -45,3 +51,17 @@ public interface Event : KordObject {
     @KordPreview
     public val customContext: Any?
 }
+
+/**
+ * Representation of an event that has not been documented / added to Kord yet.
+ *
+ * @property data the raw [json body][JsonElement] of this event if provided
+ * @property name the name of the event if provided
+ */
+public data class UnknownEvent(
+    override val kord: Kord,
+    override val shard: Int,
+    override val customContext: Any?,
+    val data: JsonElement?,
+    val name: String?
```

Comment:
i think `name` should be the first and `data` the second parameter an they should be put before `kord`, `shard` and `customContext` to match other events